corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -248,6 +249,35 @@ def _get_forms_for_user(domain, user_id, state, ids_only=False):
             return [form.form_id for form in forms]
         return forms
 
+    @staticmethod
+    def get_all_forms_received_since(received_on_since=None, chunk_size=500):
+        start_from = received_on_since or datetime.min
```

Comment:
yeah i think it seemed a little weird to be updating the function param inside the loop. no strong feelings, but happy to change